lesson_02/routes/app.route.js

```diff
@@ -0,0 +1,63 @@
+import { Router } from 'express';
+import { addUserSchema, updateUserSchema, removeUserSchema } from '../schemas/user.schema';
+import { getLoginMatch, handleError } from '../utils';
+
+export const router = Router();
+
+const store = {};
+
+router.get('/list', (req, res) =>  {
+    try {
+        res.status(201).json({ message: 'User list loaded', users: Object.values(store) });
+    } catch (e) {
+        handleError(res, e);
+    }
+});
+
+router.post('/', async (req, res) =>  {
+    try {
+        const { id, login } = req.body;
```

Comment:
id generation must be on the server side

Reply:
Done